tasks/configure_plugin.yml

```diff
@@ -0,0 +1,30 @@
+---
+- name: Create tmp file
+  tempfile:
+    state: file
+  register: tmp_file
+  changed_when: False
+
+- name: Set owner for tmp file
+  file:
+    path: "{{ tmp_file.path }}"
+    owner: www-data
+    group: www-data
+    mode: 0600
+  changed_when: False
+
+- name: "Configure {{ plugin.name }} and merge options"
+  include_tasks: configure_option_merge.yml
+  when: "{{ plugin.options }} is defined and {{ plugin.merge }} | default(wordpress_plugin_merge)"
```

Comment:
Is the first condition even needed? I don't think ansible would iterate over an empty dict.

Reply:
plugin can be set but not options so before you call it you should make sure its there

Reply:
Yes, but Ansible won't include the task file if `plugin.options` is empty?